charts/et-ccd-definitions-englandwales/Chart.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v2
+description: ET CCD Definitions England/Wales
+name: et-ccd-definitions-englandwales
+version: 0.0.1
+dependencies:
+  - name: nodejs
+    version: 2.4.1
+    repository: '@hmctspublic'
```

Comment:
I'd recommend changing all the repo dependencies in the chart to fully qualified to avoid issues in later environments like 'perftest' and 'prod'. I had an AKS helm chart issue platops asked to change to fix this by replacing the alias with 'https://hmctspublic.azurecr.io/helm/v1/repo/' on all dependencies.

Reply:
Thanks. Updated